src/foolscap/test/test_copyable.py

```diff
@@ -43,7 +43,7 @@ def slice(self, streamable, banana):
         yield 'copyable'
         yield "MyCopyable3name"
         state = self.obj.getAlternateCopyableState()
-        for k,v in state.iteritems():
+        for k,v in state.items():
```

Comment:
Note to self: extra change

Reply:
Same here. Keep it ? Have to make that change anyway otherwise.

Reply:
Ok. Resolving.